public/electron.js

```diff
@@ -201,7 +205,7 @@ ipc.on('data', (event, args) => {
     //write the data
     stream.write(JSON.stringify({...args, git}))
 
-    // Copy provocation images to patient's data folder
+    // Copy provocation images to participant's data folder
```

Comment:
this is unrelated to your changes, but line 209 and this comment make no sense for this task - can you delete them?